src/parser.rs

```diff
@@ -173,10 +153,19 @@ impl<'a> Parser<'a> {
     fn skip_unused(&mut self) {
         while self.idx < self.buf.len() {
             let c = self.buf.get(self.idx).unwrap();
-            if !matches!(c, b'\n' | b' ' | b'\r' | b'\t') {
-                break;
+            if matches!(c, b' ' | b'\n' | b'\r' | b'\t') {
+                self.step();
+                continue;
             }
-            self.step();
+            // Allow parse escaped white space
+            if *c == b'\\' && self.idx + 1 < self.buf.len() {
+                let c1 = self.buf.get(self.idx + 1).unwrap();
+                if matches!(c1, b'n' | b'r' | b't') {
```

Comment:
.

Reply:
this is escaped white space, need to check by character.